release/_promotion.sh

```diff
@@ -44,6 +46,34 @@ function prepare_poms_for_promotion {
 	sed -i "s#<version>${_ARTIFACT_RC_VERSION}</version>#<version>${_PRODUCT_VERSION}</version>#" ./*.pom
 }
 
+function promote_boms {
+	lc_time_run prepare_poms_for_promotion ${1}
+
+	lc_time_run prepare_jars_for_promotion ${1}
+
+	lc_time_run upload_boms liferay-public-releases
+}
+
+function promote_packages {
```

Comment:
@lucasmiranda0, can you please include these functions based on the alphabetic order?